src/cryptography/convert.js

```diff
@@ -44,7 +44,17 @@ export const getFirstEightBytesReversed = publicKeyBytes =>
 		.slice(0, 8)
 		.reverse();
 
-export const toAddress = buffer => `${bufferToBigNumberString(buffer)}L`;
+export const toAddress = buffer => {
+	if (
+		!Buffer.from(buffer)
+			.slice(0, 8)
+			.equals(buffer)
```

Comment:
This will break if input is an `ArrayBuffer`. Do we want that?

Reply:
Probably yes. When you input an ArrayBuffer here, this probably means you have not used getFirstEightBytesReversed before. It is likely that people end up with a wrong address. So I would say keep it strict.